test/build/config-treeshake/src/index.js

```diff
@@ -0,0 +1,8 @@
+// Test: complex comments intermixed with imports
+import def, {
+    waterfall,
+    /* map, */
+    all/* , */,
+} from 'async';
```

Comment:
nit: I _think_ this is already covered by your test, but I’d really love to test [webpack chunk name comments](https://webpack.js.org/guides/code-splitting/#dynamic-imports) as part of this test: 

```js
import(/* webpackChunkName: "lodash" */ 'lodash')
```

Reply:
Aha! You were right, we were dying on this as well. Added to test and fixed by stripping comments in the dynamic import matcher as well